src/lib.rs

```diff
@@ -266,7 +330,17 @@ impl Lzxd {
         // TODO last chunk may misalign this and on the next iteration we wouldn't be able
         // to return a continous slice. if we're called on non-aligned, we could shift things
         // and align it.
-        self.window.past_view(decoded_len)
+        
+        // Finally, postprocess the output buffer (if necessary).
+        let res = Self::postprocess(
+            self.e8_translation_size,
+            self.chunk_offset,
+            self.window.past_view(decoded_len)?,
+            &mut self.scratch_buffer,
+        );
```

Comment:
Is there any reason why this method is not called through `self.`?

Reply:
That mutable borrow against `self.scratch_buffer` prevents us from borrowing `self` - so this was a workaround.